README.md

```diff
@@ -1,24 +1,20 @@
 # ONNX Runtime Custom Ops Library
 This repository provides a library of add-on custom operators for [ONNX Runtime](http://onnxruntime.ai). The package can be installed to run with ONNX Runtime for operators not natively supported by ORT. Learn more about [custom ops in ORT](https://www.onnxruntime.ai/docs/how-to/add-custom-op.html). 
 
-# Getting started
-Windows:
-- Install Visual Studio with C++ development tools
-- Prepare Python env and install the pip packages in the requirements.txt if Python support is needed.
-- Copy build.bat to mybuild.bat and edit as needed. You may need to change "Enterprise" to "Community" depending on your Visual Studio version.
-- Run mybuild.bat
-
-Linux/MacOS:
-- Install gcc or xcode with C++ support, cmake
-- Prepare Python env and install the pip packages in the requirements.txt if Python support is needed.
-- bash ./build.sh
-
-Installation
-- cd into `out/<OS_NAME>/RelWithDebInfo` and run `pip install -e .`
+# Build and Development
+This project supports Python and build it from source easily.
+## Python package
+- Install Visual Studio with C++ development tools on Windows, or gcc for Linux or xcode for MacOS, and cmake on the unix-like platform.
+- Prepare Python env and install the pip packages in the requirements.txt
+- `python setup.py bdist_wheel` and `pip install dist/onnxruntime_customops-*.whl` to install the package.
+- OR `python setup.py develop` which is more friendly for you development, and no installation is needed.
```

Comment:
```suggestion
- OR just run `python setup.py develop` to link python to the build directory. This is more friendly for development since (re)installation is not needed with every build.
```